pkg/synthetictests/allowedalerts/types.go

```diff
@@ -0,0 +1,101 @@
+package allowedalerts
+
+import (
+	"bytes"
+	_ "embed"
+	"encoding/json"
+	"strconv"
+	"sync"
+)
+
+const (
+	p95Query = `
+SELECT * FROM openshift-ci-data-analysis.ci_data.Alerts_Unified_LastWeek_P95
+where
+  alertName = "etcdMembersDown" or 
+  alertName = "etcdGRPCRequestsSlow" or 
+  alertName = "etcdHighNumberOfFailedGRPCRequests" or 
+  alertName = "etcdMemberCommunicationSlow" or 
+  alertName = "etcdNoLeader" or 
+  alertName = "etcdHighFsyncDurations" or 
+  alertName = "etcdHighCommitDurations" or 
+  alertName = "etcdInsufficientMembers" or 
+  alertName = "etcdHighNumberOfLeaderChanges" or 
+  alertName = "KubeAPIErrorBudgetBurn" or 
+  alertName = "KubeClientErrors" or 
+  alertName = "KubePersistentVolumeErrors" or 
+  alertName = "MCDDrainError" or 
+  alertName = "PrometheusOperatorWatchErrors"
+order by 
+ AlertName, Release, FromRelease, Topology, Platform, Network
+`
+)
```

Comment:
This appears unused. How does one update the json file? We'd need that documented somewhere, and I do not know where that somewhere should be. It's a thing that someone should do periodically right? 

I'm actually a little worried about this going in hard-coded at all here, will we struggle with this getting out of date and not matching what's going on in aggregation? What are your thoughts around maintenance of this.

Pie in the sky, someday, what if this data was exposed in sippy via an unathenticated json API, and the test could hit that endpoint to fetch latest data.